tap-agent/src/agent/sender_account.rs

```diff
@@ -849,15 +837,8 @@ impl Actor for SenderAccount {
                     return Ok(());
                 };
 
-                // clean up hashset
-                state
-                    .sender_fee_tracker
-                    .unblock_allocation_id(allocation_id);
-                // update the receipt fees by reseting to 0
-                myself.cast(SenderAccountMessage::UpdateReceiptFees(
-                    allocation_id,
-                    ReceiptFees::UpdateValue(UnaggregatedReceipts::default()),
-                ))?;
+                // remove from sender_fee_tracker
+                state.sender_fee_tracker.remove(allocation_id);
```

Comment:
This is missing, to retry it, in case the sender is denied to undeny it
`// update the receipt fees by reseting to 0
myself.cast(SenderAccountMessage::UpdateReceiptFees(
    allocation_id,
    ReceiptFees::Retry,
))?;`